src/nvim/version.c

```diff
@@ -68,6 +68,7 @@ NULL
 
 // clang-format off
 static const int included_patches[] = {
+  2173,
```

Comment:
version.c updates are automated, so this can be skipped when merging vim patches.